src/stream.ts

```diff
@@ -334,6 +334,22 @@ export class LocalStream extends MediaStream {
     const track = await this.getNewTrack(kind);
     this.updateTrack(track, prev);
   }
+
+  updateMediaEncodingParams(encodingParams: RTCRtpEncodingParameters) {
+    if (!this.pc) return
+    this.pc?.getSenders()?.forEach(sender => {
```

Comment:
I don't think this should apply to every sender.  Just the one for the current stream